requirements/requirements.txt

```diff
@@ -7,14 +7,14 @@ padacioso~=0.2, >=0.2.1
 adapt-parser>=1.0.0, <2.0.0
 
 ovos-utils>=0.0.38
-ovos_bus_client<0.1.0, >=0.0.9a28
-ovos-plugin-manager<0.1.0, >=0.0.26a33
-ovos-config~=0.0,>=0.0.13a8
+ovos_bus_client~=0.1.0
```

Comment:
I dont think this works with the latest "to be stable" ovos-bus-client...https://github.com/OpenVoiceOS/ovos-bus-client/pull/107

Should be `ovos-bus-client~=0.0.9`